satpy/readers/mirs.py

```diff
@@ -256,6 +258,28 @@ def end_time(self):
             self.filename_info["end_time"] = end_time
         return self.filename_info["end_time"]
 
+    @property
+    def coeff_filenames(self):
+        """Retrieve necessary files for coefficients if needed."""
+        found_files = find_registerable_files(readers=['mirs'], writers=[],
+                                              composite_sensors=[])
+        if self.platform_name == "noaa-20":
+            platform_files = list(filter(lambda x: re.findall('noaa20', x),
+                                         found_files))
+        if self.platform_name == 'npp':
+            platform_files = list(filter(lambda x: re.findall('snpp', x),
+                                         found_files))
+        coeff_fn = {'sea': None, 'land': None}
+
+        for filename in platform_files:
+            if "atmssea" in filename:
+                coeff_fn['sea'] = retrieve(filename)
+            elif "atmsland" in filename:
+                coeff_fn['land'] = retrieve(filename)
+            else:
+                LOG.warning('Unknown coefficient filename {}', filename)
+        return coeff_fn
```

Comment:
I'd prefer if these filenames were hard-coded rather than the filter and regex stuff. You as the developer shouldn't need to use `find_registerable_files`. If you have ideas for how I can clear this up in the documentation let me know.

I was hoping it would be something like:

```
if self.platform_name == "noaa-20":
    land_fn = retrieve('limbcoef_atmsland_noaa20.txt')
coeff_fn['land'] = land_fn
```